site.py

```diff
@@ -0,0 +1,101 @@
+import os
+import subprocess
+import urllib
+import time
+import random
+from uuid import uuid4
+
+from flask import Flask, render_template, url_for, request, redirect, send_from_directory
+
+TEMP_DIR = 'tmp/'
+app = Flask(__name__)
+METADATA_FILE = 'metadata.csv'
+DATA_FILE = 'data.csv'
+
+# Confirm that a provided file has an acceptable extension.
+# In this case, '.csv' is the only allowed extension.
+def allowed_file(filename):
+    return ('.' in filename) and (filename.rsplit('.', 1)[1] in ALLOWED_EXTENSIONS)
+
+# Retrieve the files uploaded by the user, and if they are valid,
+#  store them in a new generated subfolder of tmp/.
+@app.route('/', methods=['GET', 'POST'])
+def upload_file():
+    if request.method == 'POST':
+        metadatafile = request.files['metadatafile']
+        datafile = request.files['datafile']
+        
+        if not metadatafile or not datafile:
+            render_template('index.html', missing_fields=True)
+
+        if not metadatafile.filename.endswith('.csv') 
+           or not datafile.filename.endswith('.csv'):
+            render_template('index.html', wrong_filetype=True)
+
+        print "Saving files..."
+        uniqueid = str(uuid4())
+        folder = os.path.join(TEMP_DIR, uniqueid)
+        os.mkdir(folder)
+ 
+        metadatafile.save(
```

Comment:
Remove unneeded line wrapping here and below. Only wrap if the line exceeds 79 characters.